gr.py

```diff
@@ -6,14 +6,17 @@
 import time
 import random
 import torch
-# from sd.untool import delete_runtime, free_runtime
 from model_path import model_path
-
+from sd.scheduler import samplers_k_diffusion
 DEVICE_ID = 0
 BASENAME = list(model_path.keys())
-SIZE = [("512:512", 512), ("768:768", 768)]
+SIZE = {"0": [512, 512],
+        "1": [768, 512],
+        "2": [512, 768]}
 print(BASENAME)
-scheduler = ["LCM", "DDIM"]
+scheduler = ["LCM", "DDIM", "DPM Solver++"]
```

Comment:
More schedulers are available. DPM Solver++ may need more tests. I got weird images during testing.

Reply:
ok, i tested all of scheduler in this project, and i found DPM Solver++ can not work(bad image) and DPM fast and DPM adaptive got error
```bash
sample_dpm_adaptive() got multiple values for argument 'sigma_min'
```
and
```bash
TypeError: sample_dpm_adaptive() got multiple values for argument 'sigma_min'
```
it seem we input wrong arguments in that two functions